specifications/service-registry.yaml

```diff
@@ -0,0 +1,279 @@
+    for the service providers to publish their services and the the MaaS-operators
+    and end users to query available services and the user API standards in any 
+    area of interest. The services can be filtered by used API specifications 
+    and the certification proviced by trusted third parties. The API also provides
+    methods for distributing the registered metadata between multiple instances
+    of the service.<br/>
+    <br/>
+    Copyright 2017 FLOU Ltd.
+  version: "0.1.0-oas3"
+  title: Service registry
+  contact:
+    name: FLOU
+    url: 'https://www.flou.io/'
+    email: sami.makinen@flou.io
+  license:
+    name: Apache License 2.0
+    url: 'https://github.com/maas-alliance/apis/LICENSE'
+tags:
+  - name: catalogs
+    description: Handle service catalog registration
+  - name: queries
+    description: Query registered services
+paths:
+  /catalogs/announce:
+    post:
+      tags:
+        - catalogs
+      summary: Add a new service catalog to the registry
+      description: 
+        Announce new service provider to be registered. The services area 
+        registered by providing a link to a service catalog file that specifies
+        the services and APIs provided by the service provides. The Service 
+        Registry will then update its internal database by fetching and parsing
+        the service catalog file.
+      operationId: announceCatalog
+      requestBody:
+        content:
+          application/x-www-form-urlencoded:
+            schema:
+              type: object
+              properties:
+                uri:
+                  description: Location of the service catalog resource to register
+                  type: string
+                  format: catalogUri
+              required:
+                - uri
+      responses:
+        '200':
+          description: Successful operation
+        '403':
+          description: Catalog announcement forbidden
+        '405':
+          description: Invalid input
+  /catalogs/list:
+    get:
+      tags:
+        - catalogs
+      summary: List registered service catalogs
+      description: 
+        List service catalog URIs registered with this service registry. The 
+        query can be used by other Service Registries to mirror the content of
+        another registry.
+      operationId: listCatalogs
+      responses:
+        '200':
+          description: Successful operation
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/catalog-listing'
+            application/xml:
+              schema:
+                $ref: '#/components/schemas/catalog-listing'
+        '403':
+          description: Catalog listing forbidden
+  /services/find:
+    get:
+      tags:
+        - queries
+      summary: Query registered services
+      description: Receives a list of services fulfilling the specified query criteria.
+      operationId: findServices
+      parameters:
```

Comment:
This should specify what is required, e.g., coordinate + radius, and for the non-required ones it should specify the default value.